importer/importer_utils.py

```diff
@@ -306,7 +306,15 @@ def get_text_inside_brackets(text):
     if "(" in text:
         first_bracket_index = text.find('(')
         last_bracket_index = text.rfind(')')
-        return text[first_bracket_index + 1:last_bracket_index]
+        result = text[first_bracket_index + 1:last_bracket_index]
+        if "(" in result or ")" in result:
+            for character in result:
```

Comment:
Why is "(" ok when ")" isn't?

I would probably raise ValueError on either so:
```python:
if "(" in result or ")" in result:
    raise ValueError('Nested brackets encountered')
return result
```

You should also update the docstring to clarify that the function can only handle one or less sets of brackets.